.rubocop.yml

```diff
@@ -13,7 +13,7 @@ Metrics/BlockNesting:
 # Offense count: 5
 # Configuration parameters: CountComments.
 Metrics/ClassLength:
-  Max: 1916
+  Max: 2000
```

Comment:
I agree, and this is a good idea to add to the current documentation PBI proposing how we could redesign it, and maybe not just `LanItems` but also `NetworkInterfaces` as the two main classes/modules which involves network configuration and seems to be the responsables of too many things.

BTW, thnx for the graph, I think that is very useful.